test/unit/service/jobServiceTest.ts

```diff
@@ -131,4 +131,54 @@ describe('jobService', function () {
       expect(error).to.equal('Job does not exist');
     });
   }); // end of describe getJobSpec
+
+  describe('deleteJob', function () {
```

Comment:
can you add a test to verify an error isn't returned when a successful response code is returned

Reply:
Added this test case